src/code/z_camera.c

```diff
@@ -25,6 +42,128 @@ s32 Camera_UpdateWater(Camera* camera);
 // Load the next value and scale down from camera read-only data stored in CameraModeValue
 #define GET_NEXT_SCALED_RO_DATA(values) CAM_DATA_SCALED(GET_NEXT_RO_DATA(values))
 
+#if OOT_DEBUG
+#define OREG_DEBUG(i, default) OREG(i)
+#define OREG_DEBUG_SCALED(i, default) CAM_DATA_SCALED(OREG(i))
+#else
+#define OREG_DEBUG(i, default) (default)
+#define OREG_DEBUG_SCALED(i, default) (default)
+#endif
```

Comment:
i may be misundersranding something, but cant these be removed now?

Reply:
Sorry about that